configure.ac

```diff
@@ -3991,6 +3991,21 @@ AS_VAR_IF(
    LIBMPDEC_LIBS="-lm \$(LIBMPDEC_A)"
    LIBMPDEC_INTERNAL="\$(LIBMPDEC_HEADERS) \$(LIBMPDEC_A)"])
 
+AS_VAR_IF([with_system_libmpdec], [yes],
+  [WITH_SAVE_ENV([
+    CPPFLAGS="$LIBMPDEC_CFLAGS $CPPFLAGS"
+    LDFLAGS="$LIBMPDEC_LIBS $LDFLAGS"
+    AC_SEARCH_LIBS(
+      [mpd_version], [mpdec],
+      [have_mpdec=yes], [have_mpdec=no])])],
+  [AS_VAR_SET([have_mpdec], [yes])
+   AC_MSG_WARN([m4_normalize([
+     the bundled copy of libmpdecimal is scheduled for removal in Python 3.15;
+     consider using a system installed mpdecimal library.])])])
+
+AS_IF([test "$with_system_libmpdec" = "yes" && test "$have_mpdec" = "no"],
+      [AC_MSG_WARN([no system libmpdecimal found; unable to build _decimal])])
```

Comment:
This does not have `configure` failing if `--with-system-libmpdec` is explicitly specified and there's no system library, which I thought we wanted?

Reply:
Yes, we talked about this, but I can't find any precedent for this in `configure.ac`, so I'm not sure we should go down this route. We can change that in the beta phase, though.